File: frost-scripts/performance-test/map-key-heavy-workload.py
#!/usr/bin/env python3

n = 400_000
rows = [{"k": i, "v": i * i} for i in range(n)]
result = sum(row["k"] + row["v"] for row in rows)
assert result == 21_333_333_333_200_000
print(result)

File: frost-cpp/frost-scripts/performance-test/map-key-heavy-workload.rb
#!/usr/bin/env ruby

n = 400_000
rows = (0...n).map { |i| {"k" => i, "v" => i * i} }
result = rows.sum { |row| row["k"] + row["v"] }
raise unless result == 21_333_333_333_200_000
puts result
